Chelakhovl/HW3/3.py

```diff
@@ -0,0 +1,10 @@
+#створюємо дві зміни в яких значення буде число
+
```

Comment:
good idea

but look at this version of code
a, b = b, a

Reply:
> good idea
> 
> but look at this version of code a, b = b, a

thanks, Teacher=)
I changed file with You recomendation.